### PR TITLE
update note about h2spec

### PR DESCRIPTION
**Pull request by pjfanning**
see https://github.com/apache/incubator-pekko-http/issues/218#issuecomment-1648476214

### CODE REVIEW

**Review by lomigmegard — APPROVED** [review]

**Review by nvollmar — APPROVED** [review]
lgtm

**Review by mdedetrich — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ The documentation is available at
 
 Some tests for HTTP/2 compliance use [h2spec](https://github.com/summerwind/h2spec). The sbt build downloads
 pre-built binaries from the GitHub releases page for the h2spec project. These binaries are not available for all
-operating systems. In particular, we have received reports of issues with running the tests on Apple Mac Silicon
-machines.
+operating systems. Apple Mac Silicon users may need to install [Rosetta](https://support.apple.com/en-us/HT211861) if
```
Comment (mdedetrich) [inline README.md:42]: So having a Mac M1, it already comes preinstalled with Rosetta and I never had to manually install it. Not a big deal, but you can probably rephrase this to.

"h2spec on Apple Mac Sillicon relies on Rosetta which should be already preinstalled. If not you can install it manually from https://support.apple.com/en-us/HT211861"

Comment (nvollmar) [reply README.md:42]: afaik Rosetta 2 is not preinstalled on Apple Silicon Macs, but a dialog to install it will pop up the first time you try to start an x86 application

Comment (mdedetrich) [reply README.md:42]: > sure - the language used is neutral enough - it may be best to leave it as is

Actually leave it as it is, i.e. ignore my suggestion. I misremembered regarding rosetta being installed by default.
